server/src/HttpServer.hs

```diff
@@ -16,29 +16,25 @@ import qualified Data.Text.Lazy as LText
 import qualified Network.Wai as Wai
 import qualified Web.Scotty.Trans as Scotty
 
-import qualified System.Log.Raven.Types as Sentry
-
 import JwtMiddleware (jwtMiddleware)
 import Core (Core (..), EnqueueResult (..))
 import Config (Config (..))
+import Logger
```

Comment:
We only use wildcard imports in exceptional circumstances. You should use either use an unqualified import with an explicit import list, or a qualified import.
The reason for this is that otherwise it is quite hard to find out where a given symbol comes from.